### PR TITLE
Appengine set up

### LINKED ISSUE #2: Hello World appengine setup

### PR DESCRIPTION
**Pull request by micaylamendoza**
I created a pom.xml file and an appengine-web.xml file to set up the appengine and tested it with a Hello World message in the index.html file.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```
Comment (classm-sps) [inline src/main/webapp/index.html]: Perhaps "Trick Tracker" or something instead of portfolio?

**Review by classm-sps — CHANGES_REQUESTED** [review]
The target/ files are all byproducts of the build, and shouldn't be in the repo. I'd add to the existing .gitignore file so that they never get added to PRs.

File: target/tricktracker-1/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```
Comment (CubeOfCheese) [inline target/tricktracker-1/index.html]: The title should be Trick Tracker

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]
The files in target don't need to be reached by GitHub so the target directory can be added to the .gitignore

**Comment by CubeOfCheese** [comment]
Oops it looks like Matthew already addressed the target folder stuff

File: src/main/webapp/index.html (hunk 1 above)
Comment (micaylamendoza) [reply src/main/webapp/index.html]: Oh my mistake! I forgot to change that. I'll take care of that now

**Comment by CubeOfCheese** [comment]
The target files are still in the pr so you should delete them and push a commit

**Review by CubeOfCheese — APPROVED** [review]

**Review by classm-sps — APPROVED** [review]

**Pull request merged**